modules/nf-core/rtn/tna/templates/rtn_tna.r

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env Rscript
+
+# Ported to nf-core/modules with template by Jonathan Manning
+
+#' Parse out options from a string without recourse to optparse
+#'
+#' @param x Long-form argument list like --opt1 val1 --opt2 val2
+#'
+#' @return named list of options and values similar to optparse
+
+parse_args <- function(x){
+    args_list <- unlist(strsplit(x, ' ?--')[[1]])[-1]
+    args_vals <- lapply(args_list, function(x) scan(text=x, what='character', quiet = TRUE))
+
+    # Ensure the option vectors are length 2 (key/ value) to catch empty ones
+    args_vals <- lapply(args_vals, function(z){ length(z) <- 2; z})
+
+    parsed_args <- structure(lapply(args_vals, function(x) x[2]), names = lapply(args_vals, function(x) x[1]))
+    parsed_args[! is.na(parsed_args)]
+}
+
+# Load
+library("RTN")
+library("snow")
+
+################################################
+################################################
+## Pull in module inputs                      ##
+################################################
+################################################
+
+tni_object <- readRDS('tni.rds')
```

Comment:
What if `tni_object` has a different name?

Reply:
I'm gonna update the branch. Some of these things I had fixed locally but hadn't committed/pushed yet.